base/markdown/GitHub/GitHub.jl

```diff
@@ -33,7 +33,83 @@ function github_paragraph(stream::IO, md::MD, config::Config)
     return true
 end
 
-# TODO: tables
+typealias Row Vector
```

Comment:
I think it'd be best to make this explicitly `Vector{Any}`, or you risk running into subtle typing errors.